src/igvfd/audit/file_set.py

```diff
@@ -592,12 +592,21 @@ def audit_single_cell_read_names(value, system):
             for file in value['files']:
                 if file.startswith('/sequence-files/'):
                     sequence_file_object = system.get('request').embed(file)
+                    applicable_read_types = ['R1', 'R2', 'R3']  # Skip Index 1 and Index 2
+                    # Get read type
+                    illumina_read_type = sequence_file_object.get('illumina_read_type', '')
+                    print(illumina_read_type)
+                    # If no read type or I-type, skip audit
+                    if illumina_read_type not in applicable_read_types:
+                        return
```

Comment:
I think you want continue instead of return, otherwise the audit function is terminated before checking all files

Reply:
Or even just enclose everything below line 595 in an if statement